parser_library/src/compiler_options.h

```diff
@@ -25,6 +27,9 @@ struct asm_option
     std::string sysparm;
     std::string profile;
 
+    static const instruction_set_version instr_set_default = instruction_set_version::UNI;
```

Comment:
```suggestion
    static constexpr instruction_set_version instr_set_default = instruction_set_version::UNI;
```
odr issues otherwise...